flipperdevices/flipperzero-firmware
# [FL-3699] HID: Add confirmation dialogue to the remove pairing option

# What's new

- "Remove pairing" behaviour in HID app matches the behaviour in Bluetooth settings now

# Verification 

- Check that pairing gets removed
- Check that the UI flow is as expected

# Checklist (For Reviewer)
